### PR TITLE
k8s 1.24ck1 updates

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Updated components to include latest images
- All bundle refs updated
- Release notes added
- updates to Vault
- Updates for charmed operators/addons

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

### CODE REVIEW

File: templates/kubernetes/docs/cdk-addons.md [hunk 1]
```diff
@@ -15,10 +15,19 @@ toc: False
...
+components are no longer going to be updated upstream, these are being replaced by 
+specific **Operator Charms**. Such charms are deployed by Juju into Kubernetes
+itself and require some additional steps. 
 
-This page details these components, their purpose and how they may be
-enabled/disabled where applicable.
+<div class="p-notification--positive is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-notification__message">Please see the <./operator-charms"> Operator Charms</a> page for
```
Comment (minkyngkm) [inline templates/kubernetes/docs/cdk-addons.md:27]: Could you please check the url please?

```suggestion
    <p class="p-notification__message">Please see the <a href="/kubernetes/docs/operator-charms"> Operator Charms</a> page for 
```

File: navigation.yaml [hunk 2]
```diff
@@ -439,9 +439,12 @@ kubernetes:
         - title: Basic Operations
           path: /kubernetes/docs/operations
           hidden: True
-        - title: CDK Addons
+        - title: Charmed Kubernetes Addons
           path: /kubernetes/docs/cdk-addons
           hidden: True
+        - title: Howto Addons
```
Comment (minkyngkm) [inline navigation.yaml:445]:
```suggestion
        - title: How to Addons
```

File: templates/kubernetes/docs/supported-versions.md [hunk 3]
```diff
@@ -46,6 +46,7 @@ versions of the **Charmed Kubernetes** bundle are shown in the table below:
 
 | Kubernetes version | Charmed Kubernetes bundle |
 | --- | --- |
+| 1.24.x    | [charmed-kubernetes-1154]((https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.24/bundle.yaml)) |
```
Comment (minkyngkm) [inline templates/kubernetes/docs/supported-versions.md:49]:
```suggestion
| 1.24.x    | [charmed-kubernetes-1154](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.24/bundle.yaml) |
```

File: templates/kubernetes/docs/install-manual.md [hunk 4]
```diff
@@ -217,6 +217,7 @@ versions of the **Charmed Kubernetes** bundle are shown in the table below:
 
 | Kubernetes version | Charmed Kubernetes bundle                                                                                              |
 | --- |------------------------------------------------------------------------------------------------------------------------------|
+| 1.24.x    | [charmed-kubernetes-1154]((https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.24/bundle.yaml)) |
```
Comment (minkyngkm) [inline templates/kubernetes/docs/install-manual.md:220]:
```suggestion
| 1.24.x    | [charmed-kubernetes-1154](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.24/bundle.yaml) |
```

File: templates/kubernetes/docs/inclusive-naming.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ inclusive-naming changes, and take care to make non-breaking adjustments.
 Ending with the charms release 1.23, Charmed Kubernetes is replacing the charm
 `kubernetes-master` with `kubernetes-control-plane`. This charm has always
 hosted applications such as the api-server, controller-manager, proxy, and
-scheduler. Aside from [`etcd`][etcd], which is provided by a separate charm,
+scheduler. Aside from [`etcd`](etcd), which is provided by a separate charm,
```
Comment (minkyngkm) [inline templates/kubernetes/docs/inclusive-naming.md:44]: shouldn't this be wrapped in square brackets?
```suggestion
scheduler. Aside from [`etcd`][etcd], which is provided by a separate charm,
```

**Review by minkyngkm — COMMENTED** [review]
Looking mostly good @evilnick! I've just made some comments for minor things I've found!

File: templates/kubernetes/docs/cdk-addons.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/cdk-addons.md:27]: yea, sorry, works locally ;)

**Comment by evilnick** [comment]
thanks for the updates @minkyngkm

File: templates/kubernetes/docs/supported-versions.md [hunk 6]
```diff
@@ -177,7 +178,7 @@ channels:
   1.5/candidate:    1.5.5          2017-05-17    (3) 17MB -
   1.5/beta:         1.5.5          2017-05-17    (3) 17MB -
   1.5/edge:         1.5.5          2017-05-17    (3) 17MB -
-```
+
```
Comment (minkyngkm) [inline templates/kubernetes/docs/supported-versions.md:181]: ` ``` ` This is still needed here right?

**Review by minkyngkm — APPROVED** [review]
Thanks @evilnick! Just one more thing in `/supported-versions.md` and it's good to go! 👍

File: templates/kubernetes/docs/supported-versions.md (hunk 6 above)
Comment (carkod) [reply templates/kubernetes/docs/supported-versions.md:181]: I think this is actually a correction. What's below is clearly not code, but HTML content.
There are Vanilla classes that should be displayed
```
<!-- FEEDBACK -->
<div class="p-notification--information">
  <p class="p-notification__content">
    We appreciate your feedback on the documentation. You can
    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/edit/main/pages/k8s/supported-versions.md" >edit this page</a>
    or
    <a href="https://github.com/charmed-kubernetes/kubernetes-docs/issues/new" >file a bug here</a>.</p>
  </div>
</div>
```

**Pull request merged**